<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Imitator_BRG_GUI</class>
 <widget class="QWidget" name="Imitator_BRG_GUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>859</width>
    <height>354</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="btn_refresh">
          <property name="text">
           <string>Обновить</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbl_state">
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="3" column="1">
         <widget class="QDoubleSpinBox" name="dsb_DAC_DOF_xp"/>
        </item>
        <item row="2" column="2">
         <widget class="QDoubleSpinBox" name="dsb_DAC_DGAIN_xn"/>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>X+</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_8">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Сдвиг, В</string>
          </property>
         </widget>
        </item>
        <item row="3" column="4">
         <widget class="QDoubleSpinBox" name="dsb_DAC_DOF_yn"/>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>X-</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QDoubleSpinBox" name="dsb_DDS_PW_xn"/>
        </item>
        <item row="4" column="1">
         <widget class="QDoubleSpinBox" name="dsb_DDS_PW_xp"/>
        </item>
        <item row="3" column="2">
         <widget class="QDoubleSpinBox" name="dsb_DAC_DOF_xn"/>
        </item>
        <item row="3" column="3">
         <widget class="QDoubleSpinBox" name="dsb_DAC_DOF_yp"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_9">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Сдвиг фазы, град</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QDoubleSpinBox" name="dsb_DAC_DGAIN_yp"/>
        </item>
        <item row="1" column="4">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Y-</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Y+</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Уровень, В</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QDoubleSpinBox" name="dsb_DAC_DGAIN_xp"/>
        </item>
        <item row="4" column="3">
         <widget class="QDoubleSpinBox" name="dsb_DDS_PW_yp"/>
        </item>
        <item row="2" column="4">
         <widget class="QDoubleSpinBox" name="dsb_DAC_DGAIN_yn"/>
        </item>
        <item row="4" column="4">
         <widget class="QDoubleSpinBox" name="dsb_DDS_PW_yn"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Частота, Гц</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sb_DDS_TW"/>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="btn_run">
        <property name="text">
         <string>Установить</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_stop">
        <property name="text">
         <string>Стоп</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Состояние имитатора</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="4" column="2">
         <widget class="QLineEdit" name="le_imitator_temperature"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Напряжение AVSS (вход)</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLineEdit" name="le_AVCC"/>
        </item>
        <item row="2" column="2">
         <widget class="QLineEdit" name="le_AVSS"/>
        </item>
        <item row="0" column="2">
         <widget class="QLineEdit" name="le_VCC"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Напряжение BIAS (вход)</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Напряжение AVCC (вход)</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QLineEdit" name="le_BIAS"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Напряжение питания</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Температура платы имитатора (внутренняя)</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="btn_refresh_state">
        <property name="text">
         <string>Обновить</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>btn_refresh</tabstop>
  <tabstop>dsb_DAC_DGAIN_xp</tabstop>
  <tabstop>dsb_DAC_DGAIN_xn</tabstop>
  <tabstop>dsb_DAC_DGAIN_yp</tabstop>
  <tabstop>dsb_DAC_DGAIN_yn</tabstop>
  <tabstop>dsb_DAC_DOF_xp</tabstop>
  <tabstop>dsb_DAC_DOF_xn</tabstop>
  <tabstop>dsb_DAC_DOF_yp</tabstop>
  <tabstop>dsb_DAC_DOF_yn</tabstop>
  <tabstop>dsb_DDS_PW_xp</tabstop>
  <tabstop>dsb_DDS_PW_xn</tabstop>
  <tabstop>dsb_DDS_PW_yp</tabstop>
  <tabstop>dsb_DDS_PW_yn</tabstop>
  <tabstop>sb_DDS_TW</tabstop>
  <tabstop>btn_run</tabstop>
  <tabstop>btn_stop</tabstop>
  <tabstop>le_VCC</tabstop>
  <tabstop>le_AVCC</tabstop>
  <tabstop>le_AVSS</tabstop>
  <tabstop>le_BIAS</tabstop>
  <tabstop>le_imitator_temperature</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
